capellambse/model/diagram.py

```diff
@@ -97,6 +97,9 @@ def __html__(self) -> markupsafe.Markup:
             + self.name
             + markupsafe.Markup("</figcaption></figure>")
         )
+    
+    def _short_html_(self) -> markupsafe.Markup:
+        return markupsafe.Markup(f"<b>{self.name}</b> (uuid: {self.uuid if self.uuid else 'not applicable'})")
```

Comment:
> `<b>{self.name}</b>`

`self.name` needs to be escaped here, in case it contains XML special characters. (See other `_short_html_()` or `__html__()` implementations for hints.)